acme/acme/crypto_util.py

```diff
@@ -2,7 +2,7 @@
 import binascii
 import contextlib
 import logging
-import os
+import os   # pylint: disable=os-module-forbidden
```

Comment:
This disable is not supposed to be needed, since the `acme` module is whitelisted in the linter plugin (see https://github.com/certbot/certbot/pull/8450#discussion_r522377975). Do you confirm that you still need it to make pylint happy?

Reply:
No. I reverted it.